README.md

```diff
@@ -0,0 +1,41 @@
+# CommentSold PHP SDK
+
+This SDK can be used to integrate with CommentSold via their RESTful API. Documentation
+for the API can be found at https://docs.commentsold.com/api/openapi
+
+## Requirements
+PHP 5.6.0 and later.
+
+## Composer
+You can install the bindings via [Composer](http://getcomposer.org/). Run the following command:
+
+```bash
+composer require stripe/stripe-php
+```
+
+## Getting Started
+
+Before using the SDK you will need to get a `Partner ID` and `Private Key` from [CommentSold](https://commentsold.com).
+
+There are two main sections of the API. There are the global partner functions and then the shop specific functions.
+The SDK is broken up to correspond to those two scopes. Both require their own token so be mindful of which you are using where.
+
+The first step for either is to obtain a token. Tokens are typically valid for 24 hours, so you should save them locally and reuse them in order to reduce unnecessary API calls.
+
+Example global scope SDK usage:
+```php
+$tokenizer = new \CommentSold\Api\Tokenizer('my_private_key');
+$token = $tokenizer->getPartnerToken();
+
+$client = new \CommentSold\Api\GlobalClient($token);
+$oauthUrl = $client->getOauthUrl(['all'], 'https://my-return-url.com/oauth');
+```
+
+Example shop scope SDK usage:
+```php
+$tokenizer = new \CommentSold\Api\Tokenizer('my_private_key');
+$token = $tokenizer->getShopToken('my-shop');
+
+$client = new \CommentSold\Api\ShopClient('my-shop', $token);
```

Comment:
Why doesn't $token contain both, so you only need to pass in 1 and not a matching pair

Reply:
Then we'd need to make an API call to the Gateway to decode the token. I was avoiding doing that. Or I could do it and then validate. But I feel like it might end up being a lot of API calls in practice.

Reply:
I mean if token is {string: shopId, string: token}, and it gets returned from getShopToken(), then you don't need to decode it, you have it. And you don't need to pass a matching pair into new ShopClient()

Reply:
But with the token valid for 24 hours, and them storing it, it wouldn't be set from getShopToken() each time the client is instantiated.

Reply:
They would store the token object, the pair of both values

Reply:
ok, so I pass back something like `{shop_id}:{$token}`, or a serialized object or something, and let them store that then split and use the components. It makes it less portable if they were to use this SDK and then try use native API (or potentially another SDK unless we did this in all).

Reply:
Well, for example, if we used this internally and they also needed those catalog endpoints that we kept out of the documentation (and out of the SDK) then they would need to have the raw token.

Reply:
you'd need the raw token, plus the shop id, as usual 🤷 I mean, if we were using this at all, wouldn't we write compatible code to use the same Client, same Tokens, etc? 
But again, it's okay if you don't wanna go this way.